packages/types/src/organization.ts

```diff
@@ -71,6 +76,20 @@ export type GetMembershipsParams = {
   role?: MembershipRole[];
 } & ClerkPaginationParams;
 
+/**
+ * @experimental
+ */
+export type GetRolesParams = {
+  /**
+   * This is the starting point for your fetched results. The initial value persists between re-renders
+   */
+  initialPage?: number;
+  /**
+   * Maximum number of items returned per request. The initial value persists between re-renders
+   */
+  pageSize?: number;
+};
```

Comment:
☁️ We probably need to add a refactoring ticket to create (if it does not already exist) a `PaginationParams` type for `initialPage` and `pageSize` since it's being re-used to multiple places.

Reply:
i'll add a ticket